.github/workflows/aws-ansible.yml

```diff
@@ -51,9 +51,11 @@ jobs:
           architecture: 'x64'
 
       - name: (HELPER) Install Python Libs
+        # install jinja2<v3.1.0 to prevent error "cannot import name 'environmentfilter' from 'jinja2'"
```

Comment:
What component throws that error? Considering that we do not use jinja in our code, the component that needs it should fix its dependencies. We should report it to maintainers.

Reply:
Bumped ansible to a new version in PR58, resolving the problem.